src/AtulS/CalculateAreas.java

```diff
@@ -0,0 +1,64 @@
+//Write a program to calculate area of Rectangle, area of Triangle and area of Circle using switch statement. Make it a menu driven program
+package AtulS;
+
+import java.util.Scanner;
+import java.text.*;
+
+public class CalculateAreas {
+	int len, bdt, base, hght, radius;
+
+	private double rectangle(double len, double breadth) {
+		return len * breadth;
+	}
+
+	private double triangle(double base, double height) {
+		return (1 / 2) * (base * height);
+	}
+
+	private double circle(double radius) {
+		return 3.14 * radius * radius;
+	}
+
+	private void selectArea(int num) { // select area based on give user input
+		switch (num) {
+		case 1:
+			System.out.println(rectangle(len, bdt));
+			break;
+		case 2:
+			System.out.println(triangle(base, hght));
+			break;
+		case 3:
+			System.out.println(circle(radius));
+			break;
+		}
+	}
+
+	private void input() {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Enter 1 to calculate area of Rectangle, 2 for Triangle and 3 for Circle");
```

Comment:
Better to take input from main method 
We never take input from service method..
first we take input from user n then accordingly we done operation.